charts/catalog/templates/rbac.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: v1
```

Comment:
@pmorie @arschles @luxas @liggitt  should I make a helm chart and wrap this whole thing in it? Not sure if it's okay to unconditionally send rbac objects. Would helm fail?